build/postgres/Dockerfile

```diff
@@ -98,6 +96,13 @@ else \
 	&& ${PACKAGER} -y clean all --enablerepo="epel" ; \
 fi
 
+RUN if [ "${PG_MAJOR}" -lt "15" ]; then \
+	${PACKAGER} -y install --nodocs \
+		--disablerepo=* \
+		--enablerepo=crunchypg* \
+		$( printf '11\n'${PG_MAJOR} | sort -VC && echo timescaledb_${PG_MAJOR} ) ; \
```

Comment:
🤔 This doesn't seem quite right. We don't build PG 10 now, but `PG_MAJOR=10` would run this line and the command would be ... `$PACKAGER install # nothing?`

If we're adding `RUN` commands and `if` statements, it's probably best to do all the logic there.

```sh
if [ "${PG_MAJOR}" -ge "11" ] && [ "${PG_MAJOR}" -lt "15" ]; then
```

Reply:
Ah, and now I remember why we weren't using these before. We used to build for 9.5, 9.6, etc. `sort -V` does non-integer (version, even) comparison.

Reply:
Again, I didn't realize what the `sort -V` was doing. If I move the logic back into the original `sort`, this is a non-issue?